docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -507,7 +513,7 @@ Foreign RIDs are only present if a fragile inlining was allowed at compile time.
 **TODO:** It remains to be seen whether `READYTORUN_SECTION_METHODCALL_THUNKS` and / or
```

Comment:
`READYTORUN_SECTION_INLINING_INFO` should be renamed (multiple places in the document).

Reply:
Fixed, thanks for spotting!